valohai_cli/commands/pipeline/run/run.py

```diff
@@ -21,8 +21,9 @@
 @click.option('--commit', '-c', default=None, metavar='SHA', help='The commit to use. Defaults to the current HEAD.')
 @click.option('--title', '-c', default=None, help='The optional title of the pipeline run.')
 @click.option('--adhoc', '-a', is_flag=True, help='Upload the current state of the working directory, then run it as an ad-hoc execution.')
+@click.option('--yaml', '-y', default=None, help='The path to the configuration yaml (valohai.yaml) file to use.')
```

Comment:
I'd get rid of `-y`; it's usually an alias for `--yes` (i.e. "don't confirm").
```suggestion
@click.option('--yaml', default=None, help='The path to the configuration YAML file (valohai.yaml) file to use.')
```